src/App.vue

```diff
@@ -18,8 +26,23 @@ const traitValues = ref<number[]>([0]);
 const addTrait = () => traitValues.value.push(0);
 const removeTrait = (index: number) => (traitValues.value = traitValues.value.filter((_, idx) => idx !== index));
 
-const stats = ['Combat', 'Exploration', 'Industrial', 'Trade'];
-const statValues = ref<number[]>(Array.from({ length: stats.length }, () => 0));
+const stats = ref([
+  t('translation.combat'),
+  t('translation.exploration'),
+  t('translation.industrial'),
+  t('translation.trade'),
+]);
+
+watch(selectedLanguage, () => {
```

Comment:
Is this watcher really necessary? Or doesn't the ref not auto-update because its members aren't refs themselves?